store/tikv/gcworker/gc_worker.go

```diff
@@ -307,12 +270,41 @@ func (w *GCWorker) prepare() (bool, uint64, error) {
 	return doGC, safePoint, errors.Trace(err)
 }
 
+func (w *GCWorker) initTableValues() error {
+
+	// This is a static set of data observed in TiDB on a default install *prior* to moving tikv values to sysVars.
+	// It is no longer needed, but some examples in the wild and in manual pages use "UPDATE".
+	// This requires a row to exist for the operation to succeed. Because the ON DUPLICATE KEY UPDATE
+	// action is UPDATE variable_name = variable_name, it is a noop if the row already exists.
+
+	stmt := `INSERT HIGH_PRIORITY INTO mysql.tidb VALUES
+	('tikv_gc_enable', 'true', 'Current GC enable status'),
+	('tikv_gc_run_interval', '10m0s', 'GC run interval, at least 10m, in Go format.'),
+	('tikv_gc_life_time', '10m0s', 'All versions within life time will not be collected by GC, at least 10m, in Go format.')
+	ON DUPLICATE KEY
+	UPDATE variable_name = variable_name`
+
+	se := createSession(w.store)
+	defer se.Close()
+	_, err := se.Execute(context.Background(), stmt)
+	return err
+
+}
```

Comment:
Is this really needed, since mysql.tidb is still the backend after supporting configuring by sysvars?

Reply:
Yes. The comment + the matching docs PR shows where `UPDATE is used in the wild: https://github.com/pingcap/docs/pull/4552

Reply:
I've restored the original files for `store/tikv/gcworker/`. We can revisit this if I create a refactoring PR.